fastlane/Fastfile

```diff
@@ -37,6 +37,8 @@ USER_ENV_FILE_PATH = File.join(Dir.home, '.a8c-apps', ENV_FILE_NAME)
 import 'lib/helpers.rb'
 
 before_all do
+  setup_ci if runner.current_platform == :ios
```

Comment:
As far as I know, we are only using iOS at the moment.

But given the app is built with React Native, this will save us a warning/error if we'll ever run with `platform :android`